webtau-feature-testing/examples/scenarios/graphql/queryAndMutation.groovy

```diff
@@ -44,36 +44,21 @@ mutation complete($id: ID!) {
 '''
 
 scenario("list all tasks") {
-    http.post("/graphql", [query: listAllQuery]) {
-        statusCode.should == 200
+    graphql.execute(listAllQuery) {
         errors.should == null
-        body.data.allTasks.id.should == ["a", "b", "c"]
+        allTasks.id.should == ["a", "b", "c"]
```

Comment:
makes sense to me to allow skipping the `body.data` portion, that is similar to how you can omit the `body` for http validations. 
That said, shall we also allow the "full path", i.e., `body.data.allTasks.id` ?

Reply:
Good point, yes, I think we should.